pkg/sql/logictest/testdata/logic_test/inverted_index

```diff
@@ -395,3 +395,36 @@ query IT
 SELECT * from d where b @> '[[]]' ORDER BY a;
 ----
 26  [[], {}]
+
+statement ok
+INSERT INTO d VALUES (29,  NULL)
```

Comment:
Maybe worth doing this insert higher up to ensure that its insertion doesn't impact the results of earlier queries?